www/client/containers/Routes/index.js

```diff
@@ -25,6 +26,7 @@ class Routes extends React.Component {
       <Switch>
         <Route path='/' exact component={Home} />
         <Route path='/components' exact component={ComponentOverview} />
+        <Route path='/components/component-status' exact component={ComponentStatus} />
```

Comment:
Should this page be added to the component overview page as a tab? @aferch87, thoughts?

Reply:
Moved to the component overview page.